packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -241,6 +254,13 @@ class Auth {
 
         this.redirectURI = config.redirectURI
 
+        this.passwordlessConfig = {
+            ...config.passwordlessConfig,
+            ...(callbackURI && {
+                callbackURI: isAbsoluteUrl(callbackURI) ? callbackURI : `${baseUrl}${callbackURI}`
```

Comment:
If the callbackURI is an absolute external URL, then set as is, if it is a relative path, build the URL with `baseUrl`